wagtail/admin/templates/wagtailadmin/icons/arrow-down-big.svg

```diff
@@ -0,0 +1,4 @@
+<symbol id="arrow-down-big" viewBox="0 0 16 16">
+    <path fill="currentColor"
```

Comment:
I think the fill would be better defined in CSS if possible.

Reply:
This seems like a useful default to have icons render in the same color as whatever text context they find themselves in. For example, the menu icons' color matches its adjacent text no matter what state that text is in (normal, hover, focus, etc.), with no CSS to specify the icon's color at all.

Reply:
Done in dc12760.